test/integration/040_init_test/test_init.py

```diff
@@ -546,20 +588,12 @@ def test_postgres_init_skip_profile_setup(self, mock_prompt, mock_confirm):
         project_name = self.get_project_name()
         manager.prompt.side_effect = [
             project_name,
-            1,
```

Comment:
Thanks for cleaning up the previous test! I am fairly new to all these integration tests but wondering do you think it makes sense to try to add a step after providing the valid name to make sure it can get past that? Or are there other tests covered that already?

Reply:
There's another test `test_postgres_init_with_provided_project_name` that covers the scenario that I think you're describing: https://github.com/dbt-labs/dbt-core/blob/7f92703980f4ce8069f57f5c66a8d9c376ccd87b/test/integration/040_init_test/test_init.py#L423-L444

The test I modified uses `dbt init -s` which skips all the interactive steps and doesn't consume the extra stdin "side effects".

The removed block is a verbatim copy of a similar block in `test_postgres_init_with_provided_project_name` (quoted above) which does consume them in interactive initialization. I gather the superfluous block I removed was a harmless mistake from the PR that committed both these tests.

Reply:
Yes, that covers it perfectly!! Thanks for linking it!